### DISCUSSION TITLE
Favorites & Folders

### DISCUSSION THREAD

**Discussion by belg1o** [ente-io/ente#1039] [Enhancements]
### Feature Title: Favorites & Folders

Feature Details: 
- Ability to favorite/pin to top auth key
- Ability to group auth keys by folders / tags
- Ability to custom sort the auth list would be a very nice to have as well

Detailed Description:

/

Benefits:

UX Improvements.

Impact on Existing Features, if any:
/

**Comment by Hudzen10** [comment]
Related: #8 'Add support for groups'

**Reply by itInner** [reply]
In the case of a large number of captchas, it is difficult to find the required verification, and the grouping function can solve this problem perfectly,  Is there any channce to add this feature?
